loglens

```diff
@@ -21,5 +21,5 @@ for line in sys.stdin:
     # if there was an exception, print the stack trace too
     if "exc_info" in record:
         if record["exc_info"]:
-            for line in record["exc_info"]:
-                print(line)
+            for stack_line in record["exc_info"]:
+                print(stack_line)
```

Comment:
I'm guessing it just didn't like "line"?

Reply:
If you expand the previous code, `line` was used above for another loop variable.